test/jdk/java/lang/management/ThreadMXBean/VirtualThreads.java

```diff
@@ -27,7 +27,7 @@
  * @summary Test java.lang.management.ThreadMXBean with virtual threads
  * @modules java.base/java.lang:+open java.management
  * @library /test/lib
- * @run junit/othervm VirtualThreads
+ * @run junit/othervm/native --enable-native-access=ALL-UNNAMED VirtualThreads
```

Comment:
Can you check if the end copyright date needs to be updated on this one.

Reply:
Sorry, the copyright date has been updated.